src/config/global.env.ts

```diff
@@ -25,11 +25,6 @@ export const CARD_SERVICE_URI = "CARD_SERVICE_URI";
  */
 export const HIERARCHY_SERVICE_URI = "HIERARCHY_SERVICE_URI";
 
-/**
- * URI target for the learning-object-service
- */
-export const LEARNING_OBJECT_SERVICE_URI = "LEARNING_OBJECT_SERVICE_URI";
-
 /**
```

Comment:
This is unreleated to this PR, but I noticed we still have learning object service running in staging. Is it still used at all?